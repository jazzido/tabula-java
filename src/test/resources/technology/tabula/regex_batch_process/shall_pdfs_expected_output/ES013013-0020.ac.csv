GASTON CO ENVIRONMENTAL HEALTH,,ALVIN JOHNSON,
991 WEST HUDSON BLVD,,,
"",,208 CREST LN,
"GASTONIA, NC 28052 Courier # 06-34-01",,"BELMONT, NC 28012",
EIN: 566000300EH,,,
StarLiMS ID: ES013013-0020001 Date Collected:,01/28/13,Time Collected:,12:59 PM
Date Received:,01/30/13,Collected By:,R. Whisnant
Sample Type: Raw Sampling Point:,Well head,Well Permit #:,
Analyte,,Result,Allowable Limit,Unit Qualifier(s)
Arsenic,,< 0.005,0.010,mg/L
Barium,,0.1,2.00,mg/L
Cadmium,,< 0.001,0.005,mg/L
Calcium,,41,,mg/L
Chloride,,15.00,250,mg/L
Chromium,,< 0.01,0.10,mg/L
Copper,,< 0.05,1.3,mg/L
Fluoride,,< 0.20,4.00,mg/L
Iron,,< 0.10,0.30,mg/L
Lead,,< 0.005,0.015,mg/L
Magnesium,,11,,mg/L
Manganese,,< 0.03,0.05,mg/L
Mercury,,< 0.0005,0.002,mg/L
pH,,7.3,,N/A
Selenium,,< 0.005,0.05,mg/L
Silver,,< 0.05,0.10,mg/L
Sodium,,15.00,,mg/L
Sulfate,,10.00,250,mg/L
Total Alkalinity,,130,,mg/L
Total Hardness,,150,,mg/L
Zinc,,0.07,5.00,mg/L
Report Date:,02/19/2013,,Reported By:,Debbie Moncol
