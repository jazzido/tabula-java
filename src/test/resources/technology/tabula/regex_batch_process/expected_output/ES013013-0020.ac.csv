Arsenic,< 0.005,0.010,mg/L
Barium,0.1,2.00,mg/L
Cadmium,< 0.001,0.005,mg/L
Calcium,41,,mg/L
Chloride,15.00,250,mg/L
Chromium,< 0.01,0.10,mg/L
Copper,< 0.05,1.3,mg/L
Fluoride,< 0.20,4.00,mg/L
Iron,< 0.10,0.30,mg/L
Lead,< 0.005,0.015,mg/L
Magnesium,11,,mg/L
Manganese,< 0.03,0.05,mg/L
Mercury,< 0.0005,0.002,mg/L
pH,7.3,,N/A
Selenium,< 0.005,0.05,mg/L
Silver,< 0.05,0.10,mg/L
Sodium,15.00,,mg/L
Sulfate,10.00,250,mg/L
Total Alkalinity,130,,mg/L
Total Hardness,150,,mg/L
Zinc,0.07,5.00,mg/L
